app/display.py

```diff
@@ -0,0 +1,18 @@
+from app.book import Book
+from abc import ABC, abstractmethod
+
+
+class Display(ABC):
+    @abstractmethod
+    def display(self) -> None:
+        pass
+
+
+class DisplayConsole(Display, Book):
+    def display(self) -> None:
+        print(self.content)
+
+
+class DisplayReverse(Display, Book):
+    def display(self) -> None:
+        print(self.content[::-1])
```

Comment:
There is no need to inherit the `DisplayConsole` and `DisplayReverse` classes from the `Book` class, inheriting from `Display` is enough. Instead, pass the `Book` class instance in the `__init__` or the corresponding methods as an argument.  Rewrite inheritance here and in the other files as well.

Reply:
The mentor asked to rewrite __init__ method to be able to pass Book class instance
Something like this:

```
def __init__(self, book: Book) -> None:
        self._book = book

    def display(self) -> None:
        print(self._book.content)
```